viadot/flows/mindful_to_adls.py

```diff
@@ -42,6 +44,20 @@ def adls_bulk_upload(
         )
 
 
+@task
+def add_timestamp(files_names: List = None, sep: str = None) -> None:
+    """Add new column _viadot_downloaded_at_utc into every mindful file.
```

Comment:
Maybe change the description to `Add new _viadot_downloaded_at_utc column to each file given to a function.`.
This is because it should be a CSV file rather than a Mindful file. In this case, you don't have a name suggesting the CSV file, so maybe change it to - each file given in the function.

Reply:
done!